### PR TITLE
Added nf-test for elprep/filter

### PR DESCRIPTION
**Pull request by GallVp**
Added nf-test for elprep/filter with #6226

### CODE REVIEW

File: modules/nf-core/elprep/filter/main.nf [hunk 1]
```diff
@@ -65,6 +65,8 @@ process ELPREP_FILTER {
     def assembly_regions_cmd = get_assembly_regions ? " --assembly-regions ${prefix}.assembly_regions.igv": ""
 
     """
+    mkdir logs
```
Comment (SPPearce) [inline modules/nf-core/elprep/filter/main.nf:68]: Do the logs get emitted?

Comment (GallVp) [reply modules/nf-core/elprep/filter/main.nf:68]: Thank you!

No the logs are not emitted currently. Due to the permissions issue, the docker self hosted was failing. So I added a command to create a logs folder in the current directory. Fortunately, the tool has a `--log-path` parameter.

Comment (GallVp) [reply modules/nf-core/elprep/filter/main.nf:68]: Hi @SPPearce 

Should we emit the logs? Or is this good to go?

Comment (SPPearce) [reply modules/nf-core/elprep/filter/main.nf:68]: If they are informative at all, should probably emit them

Comment (GallVp) [reply modules/nf-core/elprep/filter/main.nf]: Good idea. That's done now.

**Review by SPPearce — APPROVED** [review]

**Pull request merged**